package.json

```diff
@@ -141,7 +141,8 @@
     "typescript": "^3.8.2"
```

Comment:
I actually want to address the `eslint-plugin-*` devDependencies above.  There seem to be many, and they also could be updated.  Would it be possible to update and also constrain the plugins to the package that uses them?

Reply:
A lot of them are shared across packages.
I would argue that hoisting them to the root is the simplest, makes them easier to upgrade and have no downsides (except the root package.json being a bit more cluttered). My eventual vision would be to make the `realm` package a sub-package like all the others and in that way simplify what we're publishing to our users.

Historically I've had issues with plugins and the `eslint` package having to be co-located in the same `node_modules` directory. I haven't verified that's the case now with NPM v7, but I suspect this might still be a limitation making it difficult to move them into the sub-packages.